sqldb/interfaces.go

```diff
@@ -334,6 +334,8 @@ func (t *TransactionExecutor[Q]) ExecTx(ctx context.Context,
 		if !ok {
 			return fmt.Errorf("expected *sql.Tx, got %T", tx)
 		}
+
+		reset()
```

Comment:
I wonder if missing this partially exacerbated the issues w/ the async payment benchmark?